### PR TITLE
Pin images to commit hash not branch name

### PR DESCRIPTION
**Pull request by SimonBiggs**
So that the image links around the web aren't dependent on a branch name.

### CODE REVIEW

**Comment by sjswerdloff** [comment]
The implication being that one someone commits a new image, they have to update the README.
Not a bad thing, but something to keep in mind

File: pyproject.toml [hunk 1]
```diff
@@ -144,10 +144,6 @@ sections = [
     "MEDPHYS", "FIRSTPARTY", "LOCALFOLDER"
 ]
 
-[tool.dephell.main]
```
Comment (sjswerdloff) [inline pyproject.toml:147]: This doesn't seem related to pinning the images.
While it's not an issue, it wasn't clear from the top level commit line

**Review by sjswerdloff — APPROVED** [review]

**Comment by SimonBiggs** [comment]
> The implication being that one someone commits a new image, they have to update the README.
> Not a bad thing, but something to keep in mind

Yup, better than before though where the image in the PRs readme wouldn't actually appear changed until it merged.

File: pyproject.toml (hunk 1 above)
Comment (SimonBiggs) [reply pyproject.toml:147]: Yeah good cool. I was doing a CTRL+F for main and master and that config popped up. It's old config that isn't doing anything. Figured a fly by removal wasn't an issue in this case.

Comment (sjswerdloff) [reply pyproject.toml:147]: It's not an issue.
But it could have been if it were accidental.
So I thought I would mention it without preventing progress

Comment (SimonBiggs) [reply pyproject.toml:147]: Yup good call.
